src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LinkLabel.Link.LinkAccessibleObjectTests.cs

```diff
@@ -0,0 +1,214 @@
+        {
+            int linksCount = 4;
+            foreach (UiaCore.NavigateDirection direction in new[] { UiaCore.NavigateDirection.PreviousSibling, UiaCore.NavigateDirection.NextSibling })
+            {
+                int increment = direction == UiaCore.NavigateDirection.NextSibling ? +1 : -1;
+
+                for (int linkIndex = 0; linkIndex < linksCount; linkIndex++)
+                {
+                    yield return new object[] { (int)direction, linkIndex, linksCount, increment };
+                }
+            }
+        }
+
+        [WinFormsTheory]
+        [MemberData(nameof(LinkAccessibleObject_FragmentNavigate_Sibling_IsExpected_TestData))]
+        public void LinkAccessibleObject_FragmentNavigate_Sibling_IsExpected(int direction, int linkIndex, int linksCount, int increment)
+        {
+            using LinkLabel linkLabel = new();
+
+            for (int index = 0; index < linksCount; index++)
+            {
+                linkLabel.Links.Add(new());
+            }
+
+            int siblingIndex = linkIndex + increment;
+
+            LinkAccessibleObject expected = siblingIndex >= 0 && siblingIndex < linkLabel.Links.Count
+                ? linkLabel.Links[siblingIndex].AccessibleObject
+                : null;
+
+            LinkAccessibleObject linkAccessibleObject = linkLabel.Links[linkIndex].AccessibleObject;
+            UiaCore.IRawElementProviderFragment actual = linkAccessibleObject.FragmentNavigate((UiaCore.NavigateDirection)direction);
+
+            Assert.Equal(expected, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsTheory]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(3)]
+        public void LinkAccessibleObject_FragmentNavigate_Parent_IsExpected(int linkIndex)
+        {
+            using LinkLabel linkLabel = new();
+
+            AccessibleObject expected = linkLabel.AccessibilityObject;
+
+            for (int index = 0; index < 4; index++)
+            {
+                linkLabel.Links.Add(new());
+            }
+
+            LinkAccessibleObject linkAccessibleObject = linkLabel.Links[linkIndex].AccessibleObject;
+            UiaCore.IRawElementProviderFragment actual = linkAccessibleObject.FragmentNavigate(UiaCore.NavigateDirection.Parent);
+
+            Assert.Equal(expected, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void LinkAccessibleObject_Parent_IsExpected_IfNoOneLinkWasAdded()
+        {
+            using LinkLabel linkLabel = new();
+            LinkAccessibleObject linkAccessibleObject = linkLabel.Links[0].AccessibleObject;
+
+            AccessibleObject expected = linkLabel.AccessibilityObject;
+            AccessibleObject actual = linkAccessibleObject.Parent;
+
+            Assert.Equal(expected, actual);
```

Comment:
@vladimir-krestov, @RussKie. Do you think it is worth creating separate variables for assertion if the code is obvious and short

```suggestion
            Assert.Equal(linkLabel.AccessibilityObject, linkAccessibleObject.Parent);
```

Reply:
Actually, for this short case we can make like you suggest. This is a matter of taste.
@RussKie, may this change affect performance if we make it for thousands of tests?

Reply:
I fixed it